### PR TITLE
[aksel.nav.no] Adjust examples of Switch

### PR DESCRIPTION
**Pull request by HalvorHaugan**
Thought I would get ahead of the doc meeting next week and do the usual example adjustments in advance ;)

### CODE REVIEW

File: aksel.nav.no/website/pages/eksempler/switch/loading.tsx [hunk 1]
```diff
@@ -15,4 +15,5 @@ export const Demo = {
 
 export const args = {
   index: 8,
+  desc: "'loading' bør bare brukes for korte lastetider. Ved lengre lastetid bør brukeren bli informert om hva som tar tid/foregår i bakgrunnen mens de venter.",
```
Comment (KenAJoh) [inline aksel.nav.no/website/pages/eksempler/switch/loading.tsx]:
```suggestion
  desc: "'loading'-prop bør bare brukes for korte lastetider. Ved lengre lastetid bør brukeren bli informert om hva som tar tid/foregår i bakgrunnen mens de venter.",
```

Comment (KenAJoh) [reply aksel.nav.no/website/pages/eksempler/switch/loading.tsx]: Since we support markdown here, should we link to loading docs?

**Review by KenAJoh — DISMISSED** [review]

File: aksel.nav.no/website/pages/eksempler/switch/loading.tsx (hunk 1 above)
Comment (HalvorHaugan) [reply aksel.nav.no/website/pages/eksempler/switch/loading.tsx]: Do you mean this? https://aksel.nav.no/komponenter/core/loader#ba471ad4389c

Comment (KenAJoh) [reply aksel.nav.no/website/pages/eksempler/switch/loading.tsx]: Yea that works 👍

**Review by KenAJoh — APPROVED** [review]

**Pull request merged**